mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -48,7 +49,17 @@ public CassandraAttachmentModule() {
                     .addColumn(CassandraAttachmentTable.TYPE, text())
                     .addColumn(CassandraAttachmentTable.SIZE, bigint())
                     .withOptions()
-                    .comment("Holds attachment for fast attachment retrieval")));
+                    .comment("Holds attachment for fast attachment retrieval")),
+            new CassandraTable(CassandraAttachmentV2Table.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraAttachmentV2Table.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraAttachmentV2Table.ID, text())
+                    .addColumn(CassandraAttachmentV2Table.BLOB_ID, text())
```

Comment:
text ?

Reply:
Idem. And here changing this is out of scope of this PR. (We don't want to reindex all blobs)